tests/integration/kubernetes/tests_common.sh

```diff
@@ -37,3 +37,32 @@ get_pod_config_dir() {
 	pod_config_dir="${BATS_TEST_DIRNAME}/runtimeclass_workloads"
 	info "k8s configured to use runtimeclass"
 }
+
+# Generate kubernetes service yaml from template
+k8s_generate_service_yaml() {
+    local service_yaml="${1}"
+    local image="${2}"
+
+    local fixtures_dir="${BATS_TEST_DIRNAME}/fixtures"
+
+    # Extract name from the file name
+    local name=$(basename "${service_yaml%.*}")
+
+    local service_yaml_template="${fixtures_dir}/service.yaml.in"
+  
+    NAME="${name}" IMAGE="${image}" RUNTIMECLASS="kata" \
+        envsubst < "${service_yaml_template}" > "${service_yaml}"
+}
+
+# Set annotation for yaml
+k8s_yaml_set_annotation() {
+  local yaml="${1}"
+  local key="${2}"
+  local value="${3}"
+ 
+  # yaml annotation key name
+  local annotation_key="spec.template.metadata.annotations.\"${key}\""
+  
+  # yq set annotations in yaml
+  "${GOPATH}/bin/yq" w -i --style=double -d1 "${yaml}" "${annotation_key}" "${value}"
+}
```

Comment:
As you're adding it, please, do the following:
* Add this on its own commit
* Make sure to use it in all tests that are using annotations
  * I think CBL mariner may take advantage of this, in general
* Only then use it as part of the test being added.